lib/performance_platform/gateway/completion_rate.rb

```diff
@@ -0,0 +1,44 @@
+class PerformancePlatform::Gateway::CompletionRate
+  def fetch_stats
+    {
+      period: 'week',
```

Comment:
Feels a little counterintuitive that this is returned and not passed in as an argument, but that's a minor point.

I'd expect the users of this gateway to be deciding over what period they want this statistic (not the other way round).

Reply:
I'd agree but since so far we don't have a need for it...